src/bp-messages/classes/class-bp-messages-message.php

```diff
@@ -105,13 +105,15 @@ public function populate( $id ) {
 			)
 		);
 
-		if ( $message['messages'][0] ) {
-			$this->id        = (int) $message['messages'][0]->id;
-			$this->thread_id = (int) $message['messages'][0]->thread_id;
-			$this->sender_id = (int) $message['messages'][0]->sender_id;
-			$this->subject   = $message['messages'][0]->subject;
-			$this->message   = $message['messages'][0]->message;
-			$this->date_sent = $message['messages'][0]->date_sent;
+		$fetched_message = ( isset( $message['messages'][0] ) && is_object( $message['messages'][0] ) ) ? $message['messages'][0] : (object) array();
+
+		if ( $fetched_message ) {
```

Comment:
@janakkaneriya007 Here check not empty and Above line no need to empty array in an object

Reply:
@janakkaneriya007 This is still not fixed